rest/rest-server/src/main/java/org/ow2/proactive_grid_cloud_portal/scheduler/SchedulerStateRest.java

```diff
@@ -641,7 +649,7 @@ public JobInfoData jobInfo(String sessionId, String jobId)
             @PathParam("jobid") String jobId)
             throws NotConnectedRestException, PermissionRestException, UnknownJobRestException {
         try {
-            Scheduler s = checkAccess(sessionId, "jobs/" + jobId + "/result/value");
+            Scheduler s = checkAccess(sessionId, PATH_JOBS + jobId + "/result/value");
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Define a constant instead of duplicating this literal "/result/value" 4 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1192)